doc/content/devices/the-things-uno/_index.md

```diff
@@ -0,0 +1,412 @@
+
+## Registering The Things UNO with {{% tts %}}
+
+It’s time to register your Things Uno with {{% tts %}}. 
+
+On the **Applications** page, select your application to go to its overview page. 
+
+Select **+ Add end device** in the bottom-right of the page.
+
+{{< figure src="AddEndDevice.png" alt="Add end device" >}}
+
+{{% tts %}} provides two options to register your end device.
+
+- From the LoRaWAN Device Repository
+- Manually
+
+Each registering option provides both **OTAA** and **ABP** activation methods. Let’s have a look at them one by one.
+
+### From the LoRaWAN Device Repository
+
+This is the easiest way to register your Things Uno with {{% tts %}}.
+
+On the **Register end device** page, select the **From the LoRaWAN Device Repository** tab, if not already selected by default.
+
+Under **Select the end device**, select the following mandatory fields.
+
+- Brand - `The Things Products`
+- Model – `The Things Uno`
+- Hardware Ver. – `1.0`
+- Firmware Ver. – `quickstart` or `abp`. We recommend you to use `quickstart` as it supports [OTAA]({{< ref "devices/abp-vs-otaa#otaa" >}}).
+- Profile (Region) – Choose `EU_863_870` or `US_902_928` to match with your board. 
+
+{{< figure src="RegisterEndDeviceRepo.png" alt="Register End Device from LoRaWAN repository" >}}
+
+Under the **Enter registration data**, select/fill the following mandatory fields.
+
+If you have selected `quickstart` from the **Firmware Ver.** in the previous step:
+
+- Frequency plan – Select `Europe 863-870 MHz (SF9 for RX2 - recommended)` for **EU_863_870** or `United States 902-928 MHz, FSB2 (used by TTN)` for **US_902_928**.
+- AppEUI – Copy the **AppEUI** from the output printed by the ***TheThingsUnoTest*** sketch.
+- DevEUI – Copy the **DevEUI** from the output printed by the ***TheThingsUnoTest*** sketch.
+- AppKey – Select **Generate** button to generate an **AppKey**.
+- End device ID – Give your device a unique human-readable [identifier]({{< ref "reference/id-eui-constraints" >}}).
+
+Select the **Register end device** button. 
+
+{{< figure src="RegisterEndDeviceRepoOTAA.png" alt="Register End Device from LoRaWAN repository OTAA" >}}
+
+If you have selected `abp` from the **Firmware Ver.** in the previous step:
+
+- Frequency plan – Select `Europe 863-870 MHz (SF9 for RX2 - recommended)` for **EU_863_870** or `United States 902-928 MHz, FSB2 (used by TTN)` for **US_902_928**.
+- Device address - Select **Generate** button.
+- AppSKey– Select **Generate** button.
+- NwkSKey– Select **Generate** button.
+- End device ID – Give your device a unique human-readable [identifier]({{< ref "reference/id-eui-constraints" >}}).
+
+Select the **Register end device** button. 
+
+{{< figure src="RegisterEndDeviceRepoABP.png" alt="Register End Device from LoRaWAN repository ABP" >}}
+
+Once registered, you will be redirected to the overview page of the newly registered device, where you can find the generated **AppKey** which we’ll need next.
+
+### Manually
```

Comment:
```suggestion
### Manually
```

Let's remove the manual section. There are other instructions for manually adding devices, this isn't necessary.

Reply:
@pradeeka7 please remove this whole "Manually" section.

Reply:
fixed